model/utilities.py

```diff
@@ -20,11 +20,7 @@ def slvs_entity_pointer(cls, name, **kwargs):
     @property
     def func(self):
         index = getattr(self, index_prop)
-        if index == -1:
-            return None
-        else:
```

Comment:
Simply removing this line and unindenting the else block improves readability alot.